### PR TITLE
ui: Testing Improvements

### PR DESCRIPTION
**Pull request by pearkes**
Still WIP but:

* Ensure the a11y audit is passing for the UI (partially so, currently)
* Add some additional acceptance tests
* Prepare for `ember test` being run in CI for the UI

cc/ @dizzyup could you take a look at the result of `ember test` on this branch currently expressing a few failure for the a11y audit?

### CODE REVIEW

File: ui/tests/helpers/login.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default function login(token?: string) {
+  window.localStorage.waypointAuthToken = token || 'default-test-token-value';
```
Comment (meirish) [inline ui/tests/helpers/login.ts:2]: Nice, do you need to clear this between tests?

File: ui/app/helpers/icon-for-component.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export function iconForComponent([component]: [string]): string {
     case 'google-cloud-run':
       return 'logo-gcp-color';
     case 'kubernetes':
-      return 'logo-kubernetes-color';
+      return 'logo-kubernetes-color-alt';
```
Comment (meirish) [inline ui/app/helpers/icon-for-component.ts:17]: I think If you upgrade structure-icons to the latest you won't need this, but that can be a follow up.

**Review by meirish — APPROVED** [review]

File: ui/tests/helpers/login.ts (hunk 1 above)
Comment (pearkes) [reply ui/tests/helpers/login.ts:2]: Yes, thanks, will todo.

File: ui/app/helpers/icon-for-component.ts (hunk 2 above)
Comment (pearkes) [reply ui/app/helpers/icon-for-component.ts:17]: Yes, thank you!

File: ui/tests/helpers/login.ts (hunk 1 above)
Comment (meirish) [reply ui/tests/helpers/login.ts:2]: mentioned this, but thought I'd add the link - you may want to look into https://github.com/kaliber5/ember-window-mock

**Pull request merged**